src/IncrementalFetchingHelper.php

```diff
@@ -86,9 +128,9 @@ public function processItem(object $item, int $rowIndex): void
             return;
         }
 
-        $newValue = $item->{$this->key} ?? null;
-        $newType = $item->{$this->key . '@odata.type'}  ?? EdmType::STRING;
+        [$newValue, $newType] = $this->getIncValueAndType($item);
 
+        // Null values are not allowed
```

Comment:
V CosmosDB Table API, moze mat stlpec rocne hodnoty, pripadne byt null, ... to pre inc. f. nepovolujeme, ... vid `inc-fetching-error-null` datadir test.